hack/tools/release/notes/main.go

```diff
@@ -117,6 +115,28 @@ func (cmd *notesCmd) run() error {
 	return generator.run()
 }
 
+func releaseTypeFromNewTag(newTagConfig string) string {
+	// Error handling can be ignored as the version has been validated in computeConfigDefaults already.
+	newTag, _ := semver.ParseTolerant(newTagConfig)
```

Comment:
One last question: is it fair to ignore the error handling here? Or should we handle it anyway?